### PR TITLE
Clean manager image tag in helm after create artifacts.

### PR DESCRIPTION
**Pull request by mbobrovskyi**
#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:
Clean manager image tag in helm after create artifacts.

#### Which issue(s) this PR fixes:

Fixes #

#### Special notes for your reviewer:
Each time after run `make artifacts` we have uncommitted values.yaml file.

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

File: charts/kueue/README.md [hunk 1]
```diff
@@ -49,7 +49,8 @@ The following table lists the configurable parameters of the kueue chart and the
 | `enablePrometheus`                                     | enable Prometheus                                      | `false`                                     |
 | `enableCertManager`                                    | enable CertManager                                     | `false`                                     |
 | `controllerManager.kubeRbacProxy.image`                | controllerManager.kubeRbacProxy's image                | `gcr.io/kubebuilder/kube-rbac-proxy:v0.8.0` |
-| `controllerManager.manager.image`                      | controllerManager.manager's image                      | `us-central1-docker.pkg.dev/k8s-staging-images/kueue/kueue:main`       |
+| `controllerManager.manager.image`                      | controllerManager.manager's image                      | `us-central1-docker.pkg.dev/k8s-staging-images/kueue/kueue` |
+| `controllerManager.manager.tag`                        | controllerManager.manager's tag                        | `main`                                      |
```
Comment (alculquicondor) [inline charts/kueue/README.md:53]: this was just missing before, right?

**Review by alculquicondor — COMMENTED** [review]
/lgtm
/approve

File: charts/kueue/README.md (hunk 1 above)
Comment (mbobrovskyi) [reply charts/kueue/README.md:53]: Yes.

**Pull request merged**